Formula/hh.rb

```diff
@@ -14,17 +14,16 @@ class Hh < Formula
 
   head do
     url "https://github.com/dvorka/hstr.git"
```

Comment:
Since the head block is now only the URL, could you use the `head` one-liner after `sha256` please : 
```
head "https://github.com/dvorka/hstr.git"
```